tests/test_to_container.py

```diff
@@ -39,19 +48,21 @@ def assert_container_with_primitives(item: Any) -> None:
 
 
 @mark.parametrize(
-    "src,ex_dict,ex_dict_config,key",
+    "src,ex_dict,ex_dict_config,ex_instantiate,key",
     [
         param(
             {"user": User(age=7, name="Bond")},
             {"user": {"name": "Bond", "age": 7}},
             {"user": User(age=7, name="Bond")},
+            {"user": User(age=7, name="Bond")},
             "user",
             id="structured-inside-dict",
         ),
         param(
             [1, User(age=7, name="Bond")],
             [1, {"name": "Bond", "age": 7}],
             [1, User(age=7, name="Bond")],
+            [1, User(age=7, name="Bond")],
```

Comment:
This test in general is a bit odd.
We are passing everything to all functions, and each one is testing just one input.
can probably be refactored.

Reply:
Refactored in https://github.com/omry/omegaconf/pull/502/commits/a17a11fcd1ebbfb5a5d42c47deea0e060dbd18a3.